app/models/video.rb

```diff
@@ -1,5 +1,12 @@
 class Video < ActiveRecord::Base
   belongs_to :category
 
-  validates_presence_of :title, :description
+  validates :title, presence: true, case_sensitive: false 
```

Comment:
I think the case sensitive option probably has to do with uniqueness validations, which you are not using.